bin/install-site.sh

```diff
@@ -283,6 +283,22 @@ deb-src http://security.debian.org/ wheezy/updates main non-free
 deb http://http.debian.net/debian wheezy-backports main contrib non-free
 deb-src http://http.debian.net/debian wheezy-backports main contrib non-free
 EOF
+    elif [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
+    then
+        # Install the basic packages we require:
+        cat > /etc/apt/sources.list.d/mysociety-extra.list <<EOF
+    # Debian mirror to use, including contrib and non-free:
+    deb http://the.earth.li/debian/ jessie main contrib non-free
+    deb-src http://the.earth.li/debian/ jessie main contrib non-free
```

Comment:
I realise this is copied from the others, but is there a reason we don't use http.debian.net here too? @mhl You added this originally for squeeze, any issues with changing it to the redirector?